app/assets/stylesheets/variables.scss

```diff
@@ -40,6 +41,9 @@ body.night-theme {
     -webkit-filter: invert(95%);
     filter: invert(95%);
   }
+  .inner-comment code {
```

Comment:
I would rather update the `.single-comment-node .body code` selector which is already defined in `comments.scss`